docs/source/gateway.mdx

```diff
@@ -1,25 +1,46 @@
 ---
-title: Implementing the gateway
-description: In a federated supergraph
+title: The graph router
+description: The entry point to your federated supergraph
 ---
 
-> 📣 **The Apollo Router** is a new high-performance graph router written in Rust. You can use the Apollo Router in place of the Node.js gateway documented here. [See the documentation.](https://www.apollographql.com/docs/router/)
+import FederationArchitecture from '../shared/diagrams/federation-architecture.mdx';
 
-After you set up at least one federation-ready [subgraph](./subgraphs/), you can configure a **gateway** to sit in front of your subgraphs. The gateway serves as the entry point to your supergraph, and it executes incoming operations across one or more of your subgraphs.
+After you set up at least one federation-ready [subgraph](./subgraphs/), you can configure a **graph router** (also known as a gateway) to sit in front of your subgraphs. The router serves as the entry point to your supergraph, and it executes incoming operations across one or more of your subgraphs:
 
-The [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway) package extends Apollo Server's functionality, enabling it to act as the gateway for a supergraph.
+<FederationArchitecture />
 
-> We recommend against running your gateway in a serverless environment (such as AWS Lambda), because schema composition increases Apollo Server's startup time. If you _do_ run your gateway in a serverless environment, set your function's timeout to at least 10 seconds to prevent unexpected errors.
+## Choosing a router library
 
-## Setup
+Apollo actively supports two options for your graph router:
 
-First, let's install the necessary packages:
+* **The Apollo Router (recommended):** This is a high-performance, precompiled Rust binary.
+    * [Go to the Apollo Router docs.](/router/quickstart/)
+    * [See the federation quickstart](./quickstart/setup/) (which uses the Apollo Router).
+* **Apollo Server:** Apollo Server can act as your graph router via the [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway) extension library.
+
+**We recommend starting with the Apollo Router.** It's faster to configure, it's more performant (especially with high request loads), and it rarely requires writing custom code.
```

Comment:
No change requested, simply identifying this looks like a good place to provide a link to the new RHAI scripting available for the Router. This will enable folks to write custom code without having to know Rust.

Reply:
Good call, I'll add a link!